hw6.rb

```diff
@@ -0,0 +1,178 @@
+#   probleme 1
+#  Use the each method of Array to
+#  iterate over [1,2,3,4,5,6,7,8,9,10],
+# and print out each value.
+
+arr = [1,2,3,4,5,6,7,8,9,10]
+#  one line version
+arr.each {|el| puts el}
+
+# multi-line version
+ arr.each do |element|
+   puts element
+ end
+
+#   problem 2
+# Same as above, but only print out values greater than 5.
+# one line version
+arr.each {|element| puts element if element>= 5}
+
+# multi-line version
+arr.each do |number|
+  while number >= 5
```

Comment:
You don't need a `while` loop here, especially if you're `breaking` it unconditionally almost right after you start it)
Just `puts element if element > 5`